.github/workflows/code-lint.yml

```
@@ -28,3 +28,20 @@ jobs:
         . ./ci/common.sh
         export_or_prefix
         make lint
+
+  sc-lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.4.0
+        with:
+          submodules: true
+
+      - name: Shellcheck code
+        run: |
+          scversion="latest"
+          wget -O- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.x86_64.tar.xz" | tar -xJv
```

```suggestion
          wget -O- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.x86_64.tar.xz" | tar -xJv
```
Why use `-J` here ?

Can it replace with `-z` ?

---

Because it's format is `tar.xz` not `tar.gz`

- `-J` filter the archive through xz
- `-z` filter the archive through gzip